docs/.nvmrc

```diff
@@ -1 +1 @@
-14
```

Comment:
Do we need this file here? Shouldn't we just have a single nvmrc at the repo root?

Reply:
Each release group/independent package has its own.

Reply:
This lets us have different settings per release group, which can be useful for settings like peer dependency strictness.

Reply:
Oops, I thought this was referring to a different file. You're right that we can delete this.